Cargo.toml

```diff
@@ -64,6 +64,9 @@ indoc = "2.0"
 uzers = "0.11"
 nix = { version = "0.27", features = ["fs", "user"] }
 
+[target.'cfg(windows)'.dependencies]
+windows-projfs = { version = "0.1.6", features = ["dynamic-import"] }
```

Comment:
Does this pull in a lot of stuff? Maybe it should be optional even on Windows?

Reply:
What do you mean by "pull in a lot of stuff"?  
Do you mean memory wise, dependency wise, file wise, etc?